### PR TITLE
Add docs for bluecurrent integration

### PR DESCRIPTION
**Pull request by Floris272**
## Proposed change

Add documentation for the bluecurrent integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: #https://github.com/home-assistant/core/pull/82483
- Link to parent pull request in the Brands repository: #https://github.com/home-assistant/brands/pull/3872
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/bluecurrent.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+2. goto settings and enable developer mode
+3. Generate an api token
+
+{% include integrations/config_flow.md %}
+
+## Sensor
+
+The Blue Current integration provides the following sensors:
+<br>
+<br>
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
```

File: source/_integrations/bluecurrent.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+[BlueCurrent](https://www.bluecurrent.nl/) is an Dutch company that makes electric car chargers.
+
+The Blue Current integration allows you to connect to your blue current account to Home Assistant and monitor your chargepoint(s).
+
+## Prerequisites
+
+1. Login to the [Blue Current driver portal](https://bo.bluecurrent.nl/)
+2. goto settings and enable developer mode
+3. Generate an api token
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
3. Generate an API token.
```

File: source/_integrations/bluecurrent.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+[BlueCurrent](https://www.bluecurrent.nl/) is an Dutch company that makes electric car chargers.
+
+The Blue Current integration allows you to connect to your blue current account to Home Assistant and monitor your chargepoint(s).
+
+## Prerequisites
+
+1. Login to the [Blue Current driver portal](https://bo.bluecurrent.nl/)
+2. goto settings and enable developer mode
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
2. Goto settings and enable developer mode.
```

File: source/_integrations/bluecurrent.markdown [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  - sensor
+---
+
+[BlueCurrent](https://www.bluecurrent.nl/) is an Dutch company that makes electric car chargers.
+
+The Blue Current integration allows you to connect to your blue current account to Home Assistant and monitor your chargepoint(s).
+
+## Prerequisites
+
+1. Login to the [Blue Current driver portal](https://bo.bluecurrent.nl/)
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
1. Log in to the [Blue Current driver portal](https://bo.bluecurrent.nl/).
```

File: source/_integrations/bluecurrent.markdown [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+## Sensor
+
+The Blue Current integration provides the following sensors:
+<br>
+<br>
+
+### chargepoint sensors
+
+---
+
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
```

File: source/_integrations/bluecurrent.markdown [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+disabled by default:
+
+- current phase 1-3
+- voltage phase 1-3
+- smart charging max usage
+- offline max usage
+- remaining current
+
+<br>
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
```

File: source/_integrations/bluecurrent.markdown [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+{% include integrations/config_flow.md %}
+
+## Sensor
+
+The Blue Current integration provides the following sensors:
+<br>
+<br>
+
+### chargepoint sensors
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
### Chargepoint sensors
```

File: source/_integrations/bluecurrent.markdown [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+- current phase 1-3
+- voltage phase 1-3
+- smart charging max usage
+- offline max usage
+- remaining current
+
+<br>
+
+### grid sensors
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
### Grid sensors
```

File: source/_integrations/bluecurrent.markdown [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+---
+
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
```

File: source/_integrations/bluecurrent.markdown [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+- total kW (estimate)
+- energy usage in kWh
+- started on
+- stopped on
+- offline since
+- total cost in EUR
+- vehicle status
+- activity
+- max usage in Amps
+  - the max amps the charge point can use.
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]: Let's capatalize this list properly and sort it alphabetically.

File: source/_integrations/bluecurrent.markdown [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+disabled by default:
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
The following sensors are created as well, but disabled by default:
```

File: source/_integrations/bluecurrent.markdown [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+- current phase 1-3
+- voltage phase 1-3
+- smart charging max usage
+- offline max usage
+- remaining current
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]: Let's capitalize this list properly and sort it alphabetically.

File: source/_integrations/bluecurrent.markdown [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+<br>
+
+### grid sensors
+
+---
+
+- grid max current
+- grid average current
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]: Let's capitalize this list properly and sort it alphabetically.

File: source/_integrations/bluecurrent.markdown [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+
+disabled by default
+
+- grid current phase 1-3
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]: Let's capitalize this list properly.

File: source/_integrations/bluecurrent.markdown [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+<br>
+
+### grid sensors
+
+---
+
+- grid max current
+- grid average current
+
+disabled by default
```
Comment (frenck) [inline source/_integrations/bluecurrent.markdown]:
```suggestion
The following sensor is created as well, but disabled by default:
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/bluecurrent.markdown (hunk 1 above)
Comment (Floris272) [reply source/_integrations/bluecurrent.markdown]: Removed <br> tags

File: source/_integrations/bluecurrent.markdown (hunk 10 above)
Comment (Floris272) [reply source/_integrations/bluecurrent.markdown]: Changed.

File: source/_integrations/bluecurrent.markdown (hunk 7 above)
Comment (Floris272) [reply source/_integrations/bluecurrent.markdown]: Changed to `Charge point`.

File: source/_integrations/blue_current.markdown [hunk 16]
```diff
@@ -0,0 +1,66 @@
+---
+title: Blue Current
+description: Instructions on how to integrate Blue Current charge points within Home Assistant.
+ha_category:
+  - Sensor
+  - Car
+ha_release: 2022.12
```
Comment (joostlek) [inline source/_integrations/blue_current.markdown]:
```suggestion
ha_release: 2024.1
```

**Review by frenck — APPROVED** [review]
Thanks, @Floris272 👍 

../Frenck

**Pull request merged**